etcdserver/server.go

```diff
@@ -1275,6 +1275,8 @@ func (s *EtcdServer) applyConfChange(cc raftpb.ConfChange, confState *raftpb.Con
 // TODO: non-blocking snapshot
 func (s *EtcdServer) snapshot(snapi uint64, confState raftpb.ConfState) {
 	clone := s.store.Clone()
+	// commit kv to write metadata (for example: consistent index) to disk.
+	s.KV().Commit()
```

Comment:
Won't this `s.KV().Commit()` add a fdatasync to the apply path when there's a snapshot?